azure-pipelines-scheduled.yaml

```diff
@@ -0,0 +1,100 @@
+name: HMCTS Management Information ADF Self Hosted Integration Runtime Image - Scheduled
+
+schedules:
+  - cron: "0 22 * * *"
```

Comment:
10pm to give it ample time before the first ingestion trigger after midnight. But late enough to not impact any long running ingestions.